src/OSL-2.1.xml

```diff
@@ -0,0 +1,199 @@
+<spdx name="Open Software License 2.1" identifier="OSL-2.1" osi-approved="true">
+  <urls>
+    <url>http://opensource.org/licenses/OSL-2.1</url>
+    <url>http://web.archive.org/web/20050212003940/http://www.rosenlaw.com/osl21.htm</url>
+  </urls>
+  <notes>Same as version 2.0 of this license except with changes to section 10</notes>
+  <license>
+    <title>
+      <p>The Open Software Licensev. 2.1</p>
+    </title>
+    <optional>
+      <p>This Open Software License (the &quot;License&quot;) applies to any original work of authorship (the
+         &quot;Original Work&quot;) whose owner (the &quot;Licensor&quot;) has placed the following notice
+         immediately following the copyright notice for the Original Work:</p>
+      <p>Licensed under the Open Software License version 2.1</p>
+    </optional>
+    <body>
+      <list>
+        <li>
+          <b>1)</b>
+          <p>Grant of Copyright License. Licensor hereby grants You a world-wide, royalty-free, non-exclusive,
+             perpetual, sublicenseable license to do the following:</p>
+        </li>
+        <list>
```

Comment:
@myndzi - should new <list> be included within <li> tag of last item or like this?